PROJECT_PROGRESSION.md

```diff
@@ -122,6 +122,7 @@ This is an informational checklist to help projects onboard into the OpenJS Foun
   * infrastructure
   * legal/governance help
 - [ ] Publish security policy (see [PROJECT_SECURITY_REPORTING](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/PROJECT_SECURITY_REPORTING.md))
+- [ ] Work with the foundation to sign a Project Contribution Agreement (if needed)
```

Comment:
worth linking worth linking to the template/an example?

Reply:
I thought about linking to the template but I don't want folks filling this out prior to discussing if it is actually needed. For now I would prefer not to link to an example.

Reply:
The [template](https://docs.google.com/document/d/151tCHoS84PYpA2T-s7Wk6Us2Pv9tWJfa_C6XrBoHJto/edit) is very light-weight (it's the first time I see it). I think we should be as transparent as possible and link it.

Reply:
Understood, let me make a read-only copy and add in a link.